testing-modules/junit-5-basics-2/src/test/java/com/baeldung/junit/norunnablemethods/JUnit4UnitTest.java

```diff
@@ -0,0 +1,45 @@
+package com.baeldung.junit.norunnablemethods;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class JUnit4UnitTest {
+
+    private static int expected = 1;
+
+    @BeforeClass
+    public static void initialize() {
+        System.out.println("Intializes test setup");
+    }
+
+    @AfterClass
+    public static void tearDown() {
+        System.out.println("Test clean-up");
+    }
+
+    @Before
+    public void initTest() {
+        expected = 2;
+    }
+
+    @After
+    public void cleanUp() {
+        expected = 0;
```

Comment:
What is the purpose of this?

Reply:
Removed this class as it is redundant. Previously, I added this class to refer only to the import statements.